doc/sphinx/source/community/backward_compatibility.rst

```diff
@@ -0,0 +1,262 @@
+----------------------------------
+
+Stakeholders and their expectations and aims:
+
+Projects / Funders
+
+-  Aim to facilitate scientific discovery
+-  Expect deliverables, e.g. new features/recipes
+-  Expect reproducible results
+
+*Recipe users*
+
+-  Expect the recipe to work
+-  Expect the recipe to be easy to run
+-  Expect reproducible results
+-  Expect easy installation of ESMValTool
+
+*Recipe developers*
+
+-  Develop recipes
+-  Expect their recipe to keep working with every new *release* of
+   ESMValCore
+-  Expect ESMValCore bugfixes and new features to become available
+   quickly
+-  Expect reproducible results
+-  Expect easy installation of ESMValTool
+
+Core developers and *recipe maintainers*
+
+-  Fix bugs
+-  Add ESMValCore features requested by *recipe developers*
+-  Try to accommodate ESMValCore features contributed to by *recipe
+   developers*
+-  Maintain existing recipes
+-  Add new recipes
+-  Try to help (other) *recipe developers* with contributing their
+   recipe
+-  Try to make installation as easy as possible
+
+There is a tension between making new features available and keeping
+everything as is. New features facilitate scientific discovery because
+they enable *recipe developers* to do new research (e.g. analyse more
+data, new data, or perform a different analysis). Ensuring that every
+recipe ever made works with every new feature is technically a lot of
+work, more than we have funding for. Therefore we need to make sure that
+new features are added regularly, but we respect the timescale on which
+*recipe developers* work when removing outdated features. Writing a
+paper and getting it published may take up to a year, so this seems a
+good timescale for larger changes. For changes that only affect a few
+users, shorter timescales could be acceptable. It is also good to note
+that we are part of a large software ecosystem (ESMValTool currently
+depends on over 500 different software packages), so we may not always
+be able to control at what pace changes are made to the software that we
+depend upon.
+
+Two-way communication about new and removed features is needed to make
+this work. This requires active involvement from both the people
+developing the new features and the *recipe developers*. ESMValTool core
+developers and ESMValCore core developers need to make sure they clearly
+communicate changes. In the first place, this is done by writing good
+descriptions in issues and pull requests on GitHub, but some of this
+material also makes it to the changelog (where the GitHub pull requests
+are linked). It is highly recommended to communicate a relevant
+selection (e.g. important new, scheduled for removal, and removed
+features) also by other means, to ensure we reach as many people
+potentially affected as possible (see “Guidance on handling
+*backward-incompatible changes*” section below). We encourage
+involvement of *recipe developers* by organising monthly meetings where
```

Comment:
Do we really mean to encourage developers to *organize* monthly meetings or rather to *participate* in these meetings?

Reply:
Yes, I like this version much better.

Reply:
"attending" could also be a good word, to get the news you don't have to actively participate. Good spotting though!